libs/types/src/changes.rs

```diff
@@ -0,0 +1,38 @@
+use codec::{Decode, Encode, MaxEncodedLen};
+use frame_support::RuntimeDebug;
+use scale_info::TypeInfo;
+
+#[derive(Encode, Decode, Clone, Copy, PartialEq, Eq, RuntimeDebug, TypeInfo, MaxEncodedLen)]
+pub enum CfgChange {
+	Loan(LoanChange),
+	Pool(PoolChange),
+}
+
+impl From<LoanChange> for CfgChange {
+	fn from(value: LoanChange) -> Self {
+		CfgChange::Loan(value)
+	}
+}
+
+impl From<PoolChange> for CfgChange {
+	fn from(value: PoolChange) -> Self {
+		CfgChange::Pool(value)
+	}
+}
+
+#[derive(Encode, Decode, Clone, Copy, PartialEq, Eq, RuntimeDebug, TypeInfo, MaxEncodedLen)]
+pub enum LoanChange {
+	Maturity,
+	InterestRate,
+	InterestPayments,
+	PayDownSchedule,
+	ValuationMethod,
+	ProbabilityOfDefault,
+	LossGivenDefault,
+	DiscountRate,
+}
```

Comment:
Although, I am not sure, if the `ChangeGuard` needs more context. 

I am always not sure, if we wanna store the change on the guard side or on the executing side...

Reply:
WDYT?

Reply:
I am not 100-% sure either. Let's see in the next weeks. Not urgent

Reply:
I left some thoughts about this in the issue thread: #1207